.travis.yml

```diff
@@ -20,34 +18,119 @@ cache:
     - lighthouse-extension/node_modules
     - lighthouse-viewer/node_modules
     - /home/travis/.rvm/gems/
+    - "${HOME}/google-cloud-sdk/"
+before_install:
+  - openssl aes-256-cbc -K $encrypted_83706645e717_key -iv $encrypted_83706645e717_iv -in gcloud-credentials.json.enc -out gcloud-credentials.json -d
 install:
-  - yarn
-  # travis can't handle the parallel install (without caches)
+  # stop travis when PR and not node 6.9.1
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_NODE_VERSION" != "6.9.1" ]; then travis_terminate 0; fi'
   - yarn run install-all:task:windows
-before_script:
-  - gem install travis-artifacts
-  - export DISPLAY=:99.0
-  - export CHROME_PATH="$(pwd)/chrome-linux/chrome"
-  - sh -e /etc/init.d/xvfb start
-  - sleep 3 # wait for xvfb to boot
-  - yarn build-all
-script:
-  - yarn bundlesize
-  - yarn test-cli-formatting
-  - yarn test-launcher-formatting
-  - yarn lint
-  - yarn unit
-  - yarn closure
-  - yarn smoke
-  - yarn smokehouse
-  - yarn compile-devtools
+script: "echo \"do nothing\""
+jobs:
+  include:
+    - &prep-cache
```

Comment:
I had no idea this was a [thing](http://www.yaml.org/spec/1.2/spec.html#id2760395) that's amazing 😱 🎉

Reply:
@see https://docs.travis-ci.com/user/build-stages/using-yaml-aliases/